modules/virt-cnv-network-alerts.adoc

```diff
@@ -0,0 +1,183 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-events.html/virt-virtualization-alerts.adoc
+
+[id="virt-cnv-network-alerts_{context}"]
+= Network Alerts
+
+There are several alerts for the CNV Network operator.
+
+//CnaoDown Alert
+== CnaoDown alert
+
+.Meaning
+
+The cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster. This alert occurs when the CNAO operator is down.
+
+.Impact
+
+If the CNAO is down, changes in the CNV components are not reconciled and might not be properly deployed.
+
+.Diagnosis
+
+* Check CNAO’s operator pod namespace.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get deployment -A | grep cluster-network-addons-operator | awk '{print $1}')"
```

Comment:
@oshoval I believe we can drop this command lines 23-27 and replace $NAMESPACE with openshift-cnv. Do you agree?

Reply:
We can, but what about the following:

export NAMESPACE=openshift-cnv
The default downstream namespace is `openshift-cnv`.
It can be verified by the command ...

This will support potential cases where the namespace is changed.
Also this whole file looks like a command tips, which are connected to each other

Reply:
@oshoval  Yes, do we allow to deploy CNV on a different namespace?

Reply:
@mgarrellRH Lets keep this as is for safety.

Reply:
> @oshoval Yes, do we allow to deploy CNV on a different namespace?

Yes, the code allows arbitrary namespace for the CNAO operator
(for example the U/S one is different from the D/S one)

Reply:
btw you can use
`oc get deployment -A --field-selector metadata.name=cluster-network-addons-operator  -o=custom-columns=NS:.metadata.namespace --no-headers`

Reply:
@oshoval I like this change. It is a more elegant. I see that we use the awk in other runbooks and I think we should create a PR to update all . 
I think for now we can keep as is and Ill open an issue for the runbooks to fix it.
Once its fixed u/s we will fix it also d/s.